README.md

```
@@ -0,0 +1 @@
+# MirrorView
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)